addon/-private/column-tree.js

```diff
@@ -599,7 +600,10 @@ export default EmberObject.extend({
 
   destroy() {
     this.token.cancel();
-    get(this, 'root').destroy();
```

Comment:
Why was this destruction call insufficient?

Reply:
If the `root` computed property is invalidated (i.e. `columns` changes) then old value of root is lost and never destroyed.

Reply:
The new code in `destroy` isn't much different. It just avoids unnecessarily instantiating a root node if it wasn't already computed.

The main fix in this PR is to add the cleanup logic at the begining of the root CP computation.